apps/dashboard/src/contexts/configured-chains.tsx

```diff
@@ -16,7 +18,7 @@ const RECENTLY_USED_CHAIN_IDS_KEY = "tw-recently-used-chains";
 
 /**
  * holds the "supported chains" array
- * intially it is set to the defaultChains, then it is updated to the "allChains" with "modified chains" overrides
+ * initially it is set to the defaultChains, then it is updated to the "allChains" with "modified chains" overrides
```

Comment:
Would be nice to explain _when_ is it updated and _why_

Reply:
indeed, but would be even nicer to (soon) be able to delete all of this logic :)